### PR TITLE
pyroute2

### PR DESCRIPTION
**Pull request by awiouy**
Hoi!

Here is an addon for pyroute2. I hope it will be accepted.

Incidentally, what would be the best way to provide the pyroute libraries to the operating system? The easiest, i.e. sys.path.append("/storage/.kodi/addons/libs.pyroute2/lib/pyroute2"), is also fragile, because it depends on both the name of the addon, and on where OpenELEC stores the addon.

Thank you for your attention, and regards.

Anton Voyl

### CODE REVIEW

**Comment by stefansaraev** [comment]
for what is this needed

File: addons/tools/pyroute2/package.mk [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+PKG_DEPENDS_TARGET="toolchain"
+PKG_PRIORITY="optional"
+PKG_AUTORECONF="no"
+
+pre_make_target() {
+  export PYTHONXCPREFIX="$SYSROOT_PREFIX/usr"
+}
+
+make_target() {
+  python setup.py build
```
Comment (stefansaraev) [inline addons/tools/pyroute2/package.mk]: python setup.py build --cross-compile

File: addons/tools/pyroute2/package.mk [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+PKG_SITE="http://docs.pyroute2.org/"
+PKG_URL="https://github.com/svinota/$PKG_NAME/releases/download/$PKG_VERSION/$PKG_NAME-$PKG_VERSION.tar.gz"
+
+PKG_SHORTDESC="pyroute2, a pure Python netlink and Linux network configuration library"
+PKG_LONGDESC="pyroute2 is a pure Python netlink and Linux network configuration library. It requires only Python stdlib, no 3rd party libraries. Later it can change, but the deps tree will remain as simple, as it is possible."
+PKG_DISCLAIMER="This is an unofficial addon. Do therefore not expect support from OpenELEC forum and irc channel."
+PKG_MAINTAINER="Anton Voyl (awiouy@gmail.com)"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_TYPE="xbmc.python.module"
```
Comment (stefansaraev) [inline addons/tools/pyroute2/package.mk]: add PKG_ADDON_PROVIDES=""

**Comment by lrusak** [comment]
Is this still needed/wanted?

**Comment by chewitt** [comment]
Closing due to no update from the original submitter. Thanks.

**Pull request closed without merging**